### PR TITLE
Currency: Added build-script and build_info for cxf

### PR DESCRIPTION
**Pull request by vinodk99**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by kandarpamalipeddi** [comment]
@vinodk99 
Can you please explain why tests are skipped ?

CC: @nikhil-kalbande

**Comment by kandarpamalipeddi** [comment]
> > @vinodk99 Can you please explain why tests were skipped ?
> > CC: @nikhil-kalbande
> 
> Tried to test in power and also in intel ,tests were parity so skipped the tests and also followed the community steps which mentioned to test ,attached below https://github.com/apache/cxf/blob/main/.github/workflows/codeql-analysis.yml

As far as I know, the given `mvn` command is to analyse the code.

@seth-priya , I feel it would be good if we provide build and test commands separately (probably comment the validation command). Also it is better if we attach Power and X86 logs to this PR for future reference. Any comments ?

CC: @nikhil-kalbande

**Comment by seth-priya** [comment]
Yes, I suggested this for another one of these to Vinod - but same holds for this as well - we need to analyze the test failures even though they are in parity, we might be missing some dependency or config ot there might be a version mismatch or something else @kandarpamalipeddi . @vinodk99 please don't skip tests for any package unless otherwise agreed upon and make sure to segregate the build/test command as per @kandarpamalipeddi 's guidance

**Comment by vinodk99** [comment]
The build and test logs are long, Travis check was failing due to  job exceeded the maximum log length, and has been terminated.Attached build and test logs for reference.
[cxf_logs.txt](https://github.com/ppc64le/build-scripts/files/10496630/cxf_logs.txt)

File: c/cxf/cxf_ubi_8.5.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+###!/bin/bash -e
```
Comment (asowani) [inline c/cxf/cxf_ubi_8.5.sh:1]: Hi @vinodk99 - is this change intentional?

Comment (nikhil-kalbande) [reply c/cxf/cxf_ubi_8.5.sh:1]: @kandarpamalipeddi - Can you please comments on it. Seems it is your commit.

**Review by asowani — APPROVED** [review]
Changes reviewed.

**Review by nikhil-kalbande — APPROVED** [review]
